controllers/lvm_volumegroup.go

```diff
@@ -27,7 +28,8 @@ import (
 )
 
 const (
-	lvmVGName = "lvmvg-manager"
+	lvmVGName      = "lvmvg-manager"
+	lvmvgFinalizer = "lvm.openshift.com/volumegroup"
```

Comment:
- minor: i think `openshift.io` is used more often

Reply:
done